livekit/src/room/mod.rs

```diff
@@ -248,9 +246,10 @@ pub struct Room {
 }
 
 impl Debug for Room {
-    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+    #[tokio::main]
```

Comment:
This isn't valid, this is creating a new Tokio runtime at each call to Debug/Print.
Let's not await sid here (We could use the internal state of room to see if we already received the sid and print it, otherwise print None)